include/aspect/postprocess/visualization/heating.h

```diff
@@ -37,20 +36,36 @@ namespace aspect
     {
       /**
        * A class derived from DataPostprocessor that takes an output vector
-       * and computes a variable that represents the internal heating rate at
-       * every point.
+       * and computes several scalar variables that represent the material
+       * model outputs for every point. The list of written variables can be
+       * chosen as input parameter. The principal functionality of this class
+       * is already implemented in some other visualization plugins, the
+       * purpose of this plugin is to efficiently compute all the wanted
+       * parameters in one call to the material model.
```

Comment:
This text does not talk about "heating" at all. I suppose it is not about writing *any* material model output quantities, but specifically the ones relevant to heating.